app/src/androidTest/java/com/github/se/travelpouch/MainActivityTest.kt

```diff
@@ -0,0 +1,30 @@
+package com.github.se.travelpouch
```

Comment:
Hi, I checked with the coaches and for the first sprint it's quite good and because you have to have this code quickly. I think we can merge it. We will add additional test in time